### PR TITLE
Enhanced RDoc for Integer

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Treated:
- #+
- #-
- #*
- #fdiv
- #/
- #div
- #**
- #[]
- #digits
- #upto
- #downto

### CODE REVIEW

File: numeric.c [hunk 1]
```diff
@@ -3930,6 +3930,22 @@ fix_div(VALUE x, VALUE y)
     return fix_divide(x, y, '/');
 }
 
+/*
+ * call-seq:
+ *   self / numeric -> numeric_result
+ *
+ * Performs division:
+ *
```
Comment (jeremyevans) [inline numeric.c:3938]: I know it wasn't in the original documentation, but I really think it is worth it to document that integer division truncates if the divisor is integer.  The examples show this, but I think it is important enough to mention in the description.

**Review by jeremyevans — APPROVED** [review]
Looks good.  I do think it would be good to mention explicitly in the text how `/` does integer division if given an integer argument, and non-integer division in other cases, but as this is already demonstrated by example, I leave it up to you.

File: numeric.c (hunk 1 above)
Comment (BurdetteLamar) [reply numeric.c:3938]: I've already begun changes for the next PR.  Ok to add this then?

Comment (jeremyevans) [reply numeric.c:3938]: Sure, that's fine.

**Pull request merged**